packages/client/tests/e2e/issues/studio-1128-spawn-enoent/_steps.ts

```diff
@@ -9,15 +9,20 @@ void executeSteps({
   },
 
   test: async () => {
-    process.env.DEBUG = 'prisma:studio'
+    process.env.DEBUG = 'prisma:cli:studio'
     const studio = $`pnpm exec prisma studio`
 
+    const timer = setTimeout(() => {
+      throw new Error('"prisma studio" didn\'t request to open the browser')
+    }, 300_000)
```

Comment:
```suggestion
    }, 30_000)
```

We could make it 30s, that should be reasonable?

Reply:
I think 30 seconds is a little bit too little. I could imagine studio maybe sometimes taking longer than that to start on slow CI under load — hopefully not but still not too far fetched.

This timeout does not influence the run time of the test in success case, only in failure case, so there's no reason to not make it generous enough to not be flaky.

The idea is to have proper error message instead of a mysterious timeout after half an hour, the test taking 5 minutes only if it’s broken (and being instantaneous otherwise) is fine IMO.